ISSUE_TEMPLATE/infrastructure.yaml

```diff
@@ -0,0 +1,23 @@
+name: Infrastructure
+description: Improve CI/CD for the project
+labels:
+  - infra
+body:
+  - type: textarea
+    id: background
+    attributes:
+      label: Background
+      description: |
+        What would you like to be added for CI/CD?
+        Please provide as much info as possible
+    validations:
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
```

Comment:
leave an indent for the last line of any file

Reply:
Handled in [56b9148](https://github.com/ajktown/docs/pull/77/commits/56b9148f14feaf5b4faef3746fd82a6c6c11ff73)